app/src/androidTest/java/com/github/fribourgsdp/radio/activities/QRCodeJoinGameActivityTest.kt

```diff
@@ -0,0 +1,109 @@
+package com.github.fribourgsdp.radio.activities
+
+import android.Manifest
+import android.content.Context
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.GrantPermissionRule
+import com.github.fribourgsdp.radio.R
+import com.github.fribourgsdp.radio.game.prep.LobbyActivity
+import com.github.fribourgsdp.radio.mockimplementations.QRCodeJoinGameActivity
+import com.github.fribourgsdp.radio.mockimplementations.QRCodeJoinGameActivityJoin
+import com.github.fribourgsdp.radio.mockimplementations.QRCodeJoinGameActivityJoinFail
+import org.hamcrest.Matchers
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(AndroidJUnit4::class)
+class QRCodeJoinGameActivityTest {
```

Comment:
I moved those tests into a new file so that all tests using the QR code are together.